web/client/components/widgets/widget/MobileControls.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo } from 'react';
+import Select from 'react-select';
+import { Glyphicon } from 'react-bootstrap';
+import classnames from 'classnames';
+
+import Button from "../../../components/misc/Button";
+
+const MobileControls = ({ options }) => {
```

Comment:
I would prefer a different name because even we are using this on mobile mainly it seems to have an effect also on desktop after shrinking the screen width ( eg. WidgetSelector, WidgetInlineSelector, ... )

Reply:
It sounds reasonable to change the name, I like WidgetInlineSelector, but it doesn't effect desktop, otherwise select list would be visible there on shrinking browser window.